apps/src/lib/config/genesis/transactions.rs

```diff
@@ -579,6 +452,10 @@ pub struct ValidatorAccountTx<PK> {
     pub eth_cold_key: PK,
 }
 
+impl DeriveEstablishedAddress for UnsignedValidatorAccountTx {
+    const SALT: &'static str = "validator-account-tx";
```

Comment:
Why do we need different constant salts at all? Are we trying to avoid some kind of collision? I don't really follow.